projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DataJobsSynchronizer.java

```diff
@@ -79,15 +79,11 @@ void synchronizeDataJob(
       ActualDataJobDeployment actualDataJobDeployment,
       boolean isDeploymentPresentInKubernetes) {
     if (desiredDataJobDeployment != null) {
-      boolean sendNotification =
-          true; // TODO [miroslavi] sends notification only when the deployment is initiated by the
-      // user.
       deploymentService.updateDeployment(
           dataJob,
           desiredDataJobDeployment,
           actualDataJobDeployment,
-          isDeploymentPresentInKubernetes,
-          sendNotification);
+          isDeploymentPresentInKubernetes);
```

Comment:
I missed it before, but 'isDeploymentPresentInKubernetes' doesn't appear used  . 

It's also a bit of code smell (https://martinfowler.com/bliki/FlagArgument.html) 

And also I would not expect DataJobSynchornizer or DeploymentService interfaces (public methods and attributes) to expolose Kubernetes as it's lower level implementation detail.

Reply:
What is your suggestion here?

Reply:
I don't have a suggestion. 
I was inspiring you to come with one :) 

If you cannot think of improvement now, we can try to address this later (if at all).

Reply:
Thanks, I will leave it as it is for now.